platform-validation/scripts/analyzeSystemStats.sh

```diff
@@ -0,0 +1,245 @@
+#! /bin/bash
+
+#This script is used to get Memory and CPU Usage for Provided Binaries and Containers. The script currently stores Peak and Average CPU and Memory Usage
+#Units - Memory -MB , CPU - % , Storage Size - MB
+
+#Get Location of script
+DIR="$(cd "$(dirname "$0")" && pwd)"
+STORAGE_PATH="$(realpath "${STORAGE_PATH:-$DIR}")"
+SECONDS_TO_RUN=${SECONDS_TO_RUN:-50}
+
+BINARIES="aziot-edged aziot-identityd aziot-certd aziot-keyd dockerd containerd"
+BINARYLOCATIONS="/usr/bin /usr/libexec"
+CONTAINERS="edgeHub edgeAgent SimulatedTemperatureSensor"
+
+function usage() {
+     echo "$(basename "$0") [options]"
+     echo ""
+     echo "options"
+     echo " -h,  --help                   Print this help and exit."
```

Comment:
why 50 seconds? Is it enough?

Reply:
The run time is configurable, I simply chose an arbitrary default number of seconds if no arg is provided